test/src/test/java/org/corfudb/recovery/FastSmrMapsLoaderTest.java

```diff
@@ -1173,7 +1175,32 @@ public void doNotReconstructEmptyCheckpoints() throws Exception {
         assertThat(streamTails.get(stream1)).isEqualTo(tail1);
         assertThat(streamTails.get(stream2)).isEqualTo(tail2);
         assertThat(streamTails.get(emptyStream)).isNull();
+    }
+
+    @Test
+    public void canUseCustomSerializer() throws Exception {
```

Comment:
This test is really superfluous, using a custom serializer with objects view is already tested.
This test only tests that you can invoke a setter on the FastSmrMapsLoader object.